src/main/java/core/basesyntax/service/ConvertingFileToFruitTransaction.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ConvertingFileToFruitTransaction {
```

Comment:
Consider renaming the interface to something more indicative of its role, such as 'FileToTransactionConverter' or 'TransactionParser', to improve clarity.